server/src/yourturn.js

```diff
@@ -44,15 +49,35 @@ server.use(function(req, res, next) {
     next();
 });
 
+const inMemoryStore = stores.inMemoryStore({
+    keyExpiration: moment.duration(1, 'day')
+});
+const redisStore = stores.redisStore({
+    redis,
+    key: 'unique-tokens'
+});
+const store = stores.storeWithFallback(redisStore, inMemoryStore);
+const report = metrics.report({
+    providers: {
+        'count.unique.tokens': metrics.cachedProvider(() => store.size, 0),
+        'count.inmemory.tokens': metrics.cachedProvider(() => inMemoryStore.size, 0)
+    }
+});
+
 // EXPRESS ROUTES BELOW
 server.get('/accounts/?', routes.team.accounts);
 server.get('/teams/?', routes.team.teams);
 server.get('/teams/:teamId', routes.team.team);
 server.get('/users/:userId', routes.user.detail);
 server.get('/users/:userId/teams', routes.user.teams)
 server.get('/users/:userId/accounts', routes.user.accounts);
-server.get('/latestVersions/:team', oauth, routes.kio.latestVersions);
+server.get('/latestVersions/:team', oauth, uniqueTokens(store), routes.kio.latestVersions);
```

Comment:
Interesting question where this middleware should go. If we leave it here, only users that navigate to the application overview are counted, as this endpoint isn't called anywhere else.

What's called after every login is e.g. `/tokeninfo`, `/users/:userId` or `/users/:userId/accounts`. We could add it to one or all of them, but personally I'd just use it at `/tokeninfo`.